### ISSUE TITLE
[addressbook-level1] Short guide in case anyone is stuck trying to run it but fails

### ISSUE THREAD

**Issue by QzSG** [nusCS2113-AY1819S1/forum#4]
In case anyone else have actually tried forking [nusCS2113-AY1819S1/addressbook-level1](https://github.com/nusCS2113-AY1819S1/addressbook-level1) and followed the IntelliJ import instructions under the `developer guide` & tried to run the console app following `user guide` , you might realise that you cannot actually run `AddressBook.java` (IntelliJ asks you to create Run configuration).

This is because IntelliJ does not recognise the `src` folder as the root sources folder.

**Solution**
1. Applies if you have already imported the solution following the steps in `developer guide`
Under the `Project` pane, right click the `src` folder, select `Mark Directory As` ➡️ `Sources Root`
(Or you can manually configure under `File` -> `Project Structure...` -> `Project Settings` -> `Module`)
You should now be able to Run the console app.

2. For fresh import, doing the setup stage, uncheck import of `gradle-wrapper library`, choose `Next for everything else`

Seems like this is mentioned under the latest commits to se-edu/addressbook-level1@13daa30527fb541e6e7d073ec308537d1f21d871 (which this org's fork is some commits behind of) 

Note: Commit mentions if project is accidentally imported as Gradle project, seems instead that IntelliJ found `build.gradle` and assumes an unlinked Gradle Project despite using `importing from existing sources`

_I followed the instructions exactly and got this, it might be just on my side so if anyone else can reproc it would be nice._ 

**Debug Information**

IntelliJ IDEA 2018.2.2 (Ultimate Edition)
Build #IU-182.4129.33, built on August 21, 2018
Licensed to -redacted-
Subscription is active until -redacted-
Windows 10 10.0

@okkhoy Hope this is the correct place for this

**Comment by okkhoy** [comment]
@QzSG Perfect!

> Note: Commit mentions if project is accidentally imported as Gradle project, seems instead that IntelliJ found build.gradle and assumes an unlinked Gradle Project despite using importing from existing sources

Thanks for pointing this out! You could raise a bug on the upstream repo.

**Comment by eugenepeh** [comment]
@QzSG can I check, if you `uncheck import gradle-wrapper library` during the setting up instruction, does this problem still occurs?

**Comment by QzSG** [comment]
@eugenepeh I just reimported as a new project. That particular uncheck step seems to work, src correctly recognised. Updated my top post, perhaps you can change the upstream `developer guide` to point this out?

EDIT: 
The same `Unlinked Gradle Project` popup does appear regardless of whether `import gradle-wrapper` was checked or unchecked during import. However, unchecking it gives correct expected behavior.
I will try to reproduce this issue on my laptop.

**Comment by QzSG** [comment]
@eugenepeh Just tried reproducing on my laptop, this is weird, both installations of JDK 10.02 and IntelliJ (laptop/desktop) were fresh installs with no config changes. On my laptop, following the original instructions (with `import gradle-wrapper` still checked) simply _worked_ ... Hmm.

**Debug Information**
IntelliJ IDEA 2018.2.2 (Ultimate Edition)
Build #IU-182.4129.33, built on August 21, 2018
Licensed to -redacted-
Subscription is active until -redacted-
Windows 10 10.0

**Comment by eugenepeh** [comment]
> @eugenepeh Just tried reproducing on my laptop, this is weird, both installations of JDK 10.02 and IntelliJ (laptop/desktop) were fresh installs with no config changes. On my laptop, following the original instructions (with import gradle-wrapper still checked) simply worked ... Hmm.

Yes, this problem only affects some users. My suspicion is that new users without gradle distribution will encounter this problem when importing gradle-wrapper for the first time.
